src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -283,15 +283,32 @@ default Set<? extends ModuleElement> getAllModuleElements() {
      * <p> A documentation comment of an element is a comment that
      * begins with "{@code /**}", ends with a separate
      * "<code>*&#47;</code>", and immediately precedes the element,
-     * ignoring white space.  Therefore, a documentation comment
+     * ignoring white space and annotations and end-of-line-comments ({@code "//"} comments).
```

Comment:
Turns out that simple block comments are ignored as well. Try:
```
/**
 * This is HW.
 */
/* block comment. */
// line comment
public class HW { }
```

Reply:
Will update accordingly.